policy/modules/system/authlogin.if

```diff
@@ -1305,6 +1305,30 @@ interface(`auth_domtrans_pam_console',`
 	domtrans_pattern($1, pam_console_exec_t, pam_console_t)
 ')
 
+########################################
+## <summary>
+##	Execute pam_console in the pam timestamp domain
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to transition.
+##	</summary>
+## </param>
+## <param name="role">
+##	<summary>
+##	The role to allow the PAM domain.
```

Comment:
Nit: this should probably read "allow transitioning into the PAM domain". Otherwise LGTM.